src/Foreign/Haskell/NonEmpty.agda

```diff
@@ -0,0 +1,38 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The NonEmpty type which calls out to Haskell via the FFI
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K #-}
+
+module Foreign.Haskell.NonEmpty where
```

Comment:
Is `Foreign.Haskell.NonEmpty` ok or should I use `Foreign.Haskell.List.NonEmpty`?

Reply:
I'd be in favour of closely matching the hierarchy in `Data` so `List.NonEmpty`.